x-pack/plugins/apm/ftr_e2e/cypress/support/commands.ts

```diff
@@ -84,6 +84,22 @@ Cypress.Commands.add(
   }
 );
 
+Cypress.Commands.add(
+  'updateAdvancedSettings',
+  (settings: Record<string, unknown>) => {
+    const kibanaUrl = Cypress.env('KIBANA_URL');
+    cy.request({
+      log: false,
+      method: 'POST',
+      url: `${kibanaUrl}/api/kibana/settings`,
+      body: { changes: settings },
+      headers: {
+        'kbn-xsrf': 'e2e_test',
+      },
+    });
+  }
+);
```

Comment:
Much better to do it via the api 👍 It felt like we were writing e2e tests for Advanced settings before.